web/src/components/AuthResults/AuthResults.js

```diff
@@ -13,22 +23,28 @@ export default () => {
     setLastUpdate(new Date().toLocaleTimeString())
   }, [currentUser, userMetadata, isAuthenticated])
 
-  return (
-    <div className="w-full overflow-x-auto overflow-y-auto text-sm max-h-80">
-      <p>Last update {lastUpdate}</p>
-      <code>
-        userMetaData:
-        <pre style={{ margin: 0 }}>{JSON.stringify(userMetadata, null, 2)}</pre>
-      </code>
-      <br />
-      <code>
-        currentUser:
-        <pre style={{ margin: 0 }}>{JSON.stringify(currentUser, null, 2)}</pre>
-      </code>
+  const authData = (userMetadata, currentUser) => {
+    let Metadata = 'userMetadata: ' + JSON.stringify(userMetadata, null, 2) + '\n'
+    let User = 'currentUser: ' + JSON.stringify(currentUser, null, 2)
+    return Metadata + User
+  }
 
+  return (
+    <div className="w-full text-sm">
+      <p className="mt-2 text-sm text-gray-600 max-w"> Last update {lastUpdate}</p>
+      <Codemirror
```

Comment:
Should we use CodeMirror or `highlightjs`? Redwoodjs.com uses `highlightjs`.

We're going to have code samples, and maybe these presentations should use the same package?

Reply:
Ok, if 

> CodeMirror is a code editor. 

then that doesn't seem the best tool.

I think @morganmspencer is working on the Code Samples. Once those are in, we can use the same package here to format the JSON for the User.

Reply:
@dthyresson  that makes sense, I actually am not familiar w/ highlightjs I'll take a look at it once I'm done with class.